lib/storage/src/content_manager/toc.rs

```diff
@@ -129,7 +116,7 @@ impl TableOfContent {
                 Wal::open(collections_meta_wal_path).unwrap(),
             ))
         };
-        #[cfg(feature = "consensus")]
+
```

Comment:
the raft state is now always initialized, not sure if it could be problematic TBH

Reply:
It's a small file, shouldn't really matter to user I think